members.md

```diff
@@ -1,14 +1,14 @@
 List of 2020/2021 MIPT-V project members:
 1. Kirill Korolev
-1. Pavel Kryukov
-2. Alexey Shcherbakov
-3. Anton Okley
-4. Vasilii Zaitsev
-5. Ruslan Zaripov
-6. Ivan Borisenkov
-7. Vladimir Graudt
-8. Egor Titov
-9. Eugene Naydanov
-10. Mihail Fedorov
-11. Kirill Kononovich
-12. Ravil Zakiryanov
+2. Pavel Kryukov
+3. Alexey Shcherbakov
+4. Anton Okley
+5. Vasilii Zaitsev
+6. Ruslan Zaripov
+7. Ivan Borisenkov
+8. Vladimir Graudt
+9. Egor Titov
+10. Eugene Naydanov
+11. Mihail Fedorov
+12. Kirill Kononovich
+13. Ravil Zakiryanov
```

Comment:
That change is likely to complicate merges for other participants. Could you please revert it?
Also, Markdown fixes enumerations automatically,

Reply:
Okay, i'll read how to revert it

Reply:
I propose to do the following:
1. Uncommit your changes: `git reset origin/master`
2. Do an interactive staging: `git add -p`. Then press "y" for tokens you want to keep and "n" for otherwise.
3. Commit: `git commit`. You must not use "-a" flag!
4. Eliminate unstaged changes: `git reset --hard HEAD`

Reply:
Alternative approach:
1. Create a patch: `git diff > revert.patch`
2. Open patch in editor and remove **needed** code.
3. Revert patch `git apply -R revert.patch`
4. Commit with proper message: `git commit -a -m "Take code review comments into account"`
5. Push: `git push origin master`

Reply:
I think I made a mistake somewhere. Is it normal to delete my repo and fork it again?

Reply:
Yes, if you think it is simpler.

Reply:
I get error while pulling riscvopcodes
v-opcodes master
Из https://github.com/MIPT-ILab/riscv-opcodes
 * branch              master     -> FETCH_HEAD
fatal: отказ слияния несвязанных историй изменений

Reply:
So, can i create new pull request?

Reply:
Yes.